internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1245,3 +1248,22 @@ func (c *ClusterManager) setKafkaPerClusterCountMetrics() error {
 
 	return nil
 }
+
+func (c *ClusterManager) setKasFleetManagerClusterProviderResourceQuotaConsumedMetric() error {
+	ocmCredentialsSpecified := (c.OCMConfig.ClientID != "" && c.OCMConfig.ClientSecret != "") || c.OCMConfig.SelfToken != ""
```

Comment:
This is also here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1251 maybe it is good to have a `.CheckIfCredentialsAreProvided()` method in the `OCMConfig` struct that can be used here and in the linked PR

/cc @JameelB @JasonPowr

Reply:
having a method for it sounds good. We can use it in other places too if needed in the future.

Reply:
Shouldn't we check instead whether the cluster provider is OCM first at all?

Reply:
Additionally, why is this metric limited to OCM?

Reply:
> Shouldn't we check instead whether the cluster provider is OCM instead?

These are not cluster specific metrics but cluster provider e.g OCM, wide metrics.  

> Additionally, why is this metric limited to clusters provisioned through OCM?

For other only provider that we've i.e `standalone` the count will be zero everywhere (`consumed` and `max`). I think once we've other cluster providers we could add them.